addons/test_event_full/tests/test_event_security.py

```diff
@@ -104,7 +104,6 @@ def test_event_access_event_manager(self):
         # Event Type
         event_type = self.env['event.type'].create({
             'name': 'ManagerEventType',
-            'use_mail_schedule': True,
```

Comment:
you forgot to remove this field usage at addons/test_event_full/tests/test_event_security.py:95

Reply:
Oh yeah, it has now been deleted.